src/helpers/storage.ts

```diff
@@ -0,0 +1,10 @@
+const FavoriteKey = 'favorites'
+
+export function loadFavorites(): { [key: number]: true | undefined } {
+  const res = localStorage.getItem(FavoriteKey)
+  return res === null ? {} : JSON.parse(res)
+}
```

Comment:
could be this also I think: `return JSON.parse(localStorage.getItem(FavoriteKey)) ?? {}`